docs/docs/roadmap.md

```diff
@@ -0,0 +1,50 @@
+# Roadmap 🗺️
+
+In the interest of transparency, we want to share high-level details of our roadmap, so that others can see our priorities and make plans based off the work we are doing.
+
+## Areas of Focus 💡
+
```

Comment:
What are you considering for this section?

Reply:
I just intended this to be a subheading for all of the different areas we're focusing on but I'm happy to restructure. At the moment, I just used the existing ROADMAP format from the root of the project.